extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -46,15 +66,61 @@ function render_block( $attr, $content ) {
 		true
 	);
 
+	// Show the embedded version.
+	if ( empty( $attr['useModal'] ) && ( empty( $attr['style'] ) || 'modal' !== $attr['style'] ) ) {
+		return render_embed_block( $attr );
+	} else {
+		return render_modal_block( $attr, $content );
+	}
+}
+
+/**
+ * Render block with embed style.
+ *
+ * @param array $attr Eventbrite block attributes.
+ * @return string Rendered block.
+ */
+function render_embed_block( $attr ) {
 	$widget_id = wp_unique_id( 'eventbrite-widget-' );
 
-	wp_enqueue_script( 'eventbrite-widget', 'https://www.eventbrite.com/static/widgets/eb_widgets.js', array(), JETPACK__VERSION, true );
+	$is_amp = Jetpack_AMP_Support::is_amp_request();
 
-	// Add CSS to hide direct link.
-	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
+	// $content contains a fallback link to the event that's saved in the post_content.
+	// Append a div that will hold the iframe embed created by the Eventbrite widget.js.
+	$classes = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attr );
+
+	$classes .= ' wp-block-jetpack-eventbrite--embed';
+
+	$direct_link = sprintf(
+		'<a href="%s" rel="noopener noreferrer" target="_blank" class="eventbrite__direct-link" %s>%s</a>',
+		esc_url( $attr['url'] ),
+		$is_amp ? 'placeholder fallback' : '',
+		esc_html__( 'Register on Eventbrite', 'jetpack' ),
+	);
+
+	if ( $is_amp ) {
+		$embed = sprintf(
+			'<amp-iframe src="%s" layout="responsive" resizable width="1" height="1" sandbox="allow-scripts allow-same-origin allow-forms"><button overflow>%s</button>%s</amp-iframe>',
+			esc_url(
+				add_query_arg(
+					array(
+						'eid'    => $attr['eventId'],
+						'parent' => rawurlencode( get_current_url() ),
+					),
+					'https://www.eventbrite.com/checkout-external'
+				)
+			),
+			esc_html__( 'Expand', 'jetpack' ),
+			$direct_link
+		);
+	} else {
+		$embed = $direct_link;
```

Comment:
Not sure I understand.

The purpose of the `$direct_link` is a fallback for when JS is disabled or fails to load. Previously here it was put inside of a `noscript` element, but now this is changed to put it directly as the child of the `div` and the CSS has been amended to inlude:

```css
.eventbrite__direct-link:not(:only-child) {
	display: none;
}
```

When the Eventbrite widget initializes, it appends the `div` with the `iframe` and then that causes the selector to start matching and the direct link is then hidden. This is good as at that point the link is obsolete since the iframe has been successfully constructed.